core/dbt/events/types.proto

```diff
@@ -37,6 +37,10 @@ message NodeInfo {
     string node_started_at = 7;
     string node_finished_at = 8;
     google.protobuf.Struct meta = 9;
+    string node_database = 10;
+    string node_schema = 11;
+    string node_alias = 12;
+    string node_relation_name = 13;
```

Comment:
@gshank @peterallenwebb Does either of you have a strong feeling around whether:
- We should prefix these with `node_` (like the other ones)
- We should nest these as fields in a `relation_info` dictionary, instead of having them all as top-level fields (strings)

Reply:
tbh I kind of wish none of the other fields started with "node_". It feels duplicative. But that's moot, I guess :)

Nested or not I don't have strong feelings.

I wonder if we need to make these "optional" though. Did you test this with a node with database = None?  The version of betterproto we were using before didn't have "optional", and google protobuf behaves a bit differently anyway. I'm not quite sure what it will do with a None passed in. Changing the event in test_events.py should tell.

Reply:
How about node_relation: { database='...', schema='', alias='', name: '...' ?

Reply:
If I pass in `None` for `database`, it comes back as `""`. I'll add a test for this. I think this is how proto works, right? All fields are technically optional, and there's no way to really distinguish between empty string (`""`) and missing (`None`), short of adding another boolean field `has_database`

Reply:
There is an "optional" thing, but it's not exactly the same as in Python, so I suspect we'd still get an empty string. betterproto was trying to be more python-like, so I think they implemented it in a somewhat more pythonic way, but I haven't done any optional fields with google protobuf yet...

Reply:
I went with
```
node_relation: {database='...', schema='...', alias='...', relation_name: '...'}
```
To match the nomenclature in the existing top-level node attributes, and because `name` by itself feels closer to alias/identifier